test/features/managet_test_spider.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Features.Manager.TestSpider do
```

Comment:
I like the idea of splitting this code as you're suggesting. But could we address it in a separate PR?